Cargo.toml

```
@@ -21,9 +21,14 @@ categories = ["network-programming", "data-structures"]
 [features]
 default = ["std"]
 std = []
+# Use portable-atomic crate to support platforms without atomic CAS.
+# See https://docs.rs/portable-atomic for more information.
+extra-platforms = ["portable-atomic"]
 
 [dependencies]
 serde = { version = "1.0.60", optional = true, default-features = false, features = ["alloc"] }
+# Enable require-cas feature to provide a better error message if the end user forgets to use the cfg or feature.
+portable-atomic = { version = "1.3", optional = true, default-features = false, features = ["require-cas"] }
```

The possibility of confusion is a good point. Another option is to rename this crate so that there isn't a feature called `portable-atomic`.
```suggestion
extra-platforms = { package = "portable-atomic", version = "1.3", optional = true, default-features = false, features = ["require-cas"] }
```

---

The suggested approach sounds good to me. Done in https://github.com/tokio-rs/bytes/commit/b30fe89f5212a6c54ecbffab17021e2ca538c653.